### DISCUSSION TITLE
About the process of some special character like `\\`

### DISCUSSION THREAD

**Discussion by xsro** [yuin/goldmark#243] [Q&A]
`\\` is usually used in some math environment for newline, but  the renderer transform it as `\` or `\<br>`
This cause some problem for math renderer like katex.

As I tested in hugo, following code will be rendered as `\` or `\<br>`, but I want `\\`

full test code

```
before\\

\\after

before\\after

beforeline
\\

\\
afterline

beforeline
\\
afterline
```

result

<img width="530" alt="截屏2021-08-29 下午10 04 28" src="https://user-images.githubusercontent.com/64292434/131253256-9fc552b4-8701-4b11-b1f6-545adc5fb6e5.png">

Is there a way to config how the goldmark process special charcter like this?

**Comment by Pinghigh** [comment]
Maybe none, just use `\\\\`
